TribalWars/src/ApplicationCore/Response/Response.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApplicationCore.Response
```

Comment:
Other namespaces in ApplicationCore are named by plural form, but Response is in singular form, that is a little bit inconsistent.